### ISSUE TITLE
unable to find "influxdb" stats_pusher

### ISSUE THREAD

**Issue by ghost** [unbit/uwsgi-influxdb#1]
I followed the steps in the README and when I start uwsgi I see this error in the logs and uwsgi dies.

Installation: 

```
uwsgi --build-plugin https://github.com/unbit/uwsgi-influxdb
*** uWSGI building and linking plugin from https://github.com/unbit/uwsgi-influxdb ***
Cloning into 'uwsgi-influxdb'...
remote: Counting objects: 11, done.
remote: Total 11 (delta 0), reused 0 (delta 0), pack-reused 11
Unpacking objects: 100% (11/11), done.
Checking connectivity... done.
[gcc -pthread] influxdb_plugin.so
build time: 0 seconds
*** influxdb plugin built and available in influxdb_plugin.so ***
```

uwsgi config:

```
enable-metrics = true
plugin = influxdb
stats-push = influxdb:http://influx.server:8086/db/uwsgi/series?u=root&p=root
```

uwsgi log:

`[uwsgi] 25/11/2015 15:17:45 unable to find "influxdb" stats_pusher`

**Comment by unbit** [comment]
Are you sure the "influxdb_plugin.so" library is in the current directory ?

eventually specify its absolute path in the plugin directive

**Comment by ghost** [comment]
Yup, providing the full path to the influxdb_plugin.so worked like a charm. Now I see 404 errors and I'm going to assume that's because this plugin is not 0.9+ compatible, is that correct? If that's the case, are there any plans to update it?

**Comment by unbit** [comment]
If i understand correctly the json protocol is now deprecated, but should work if you change the url entrypoint.

**Comment by ghost** [comment]
What should it change to? I'm quite new to influx so I'm not sure what I should look for so any help, deeply appreciated!

**Comment by SEJeff** [comment]
@unbit my C is quite rusty (haven't done much in the past 10 years), but I'd be happy to take a stab at porting this to use the new udp [line protocol](https://docs.influxdata.com/influxdb/v0.13/write_protocols/write_syntax/) unless you'd prefer to do it yourself? Should it be made configurable, or just moved entirely to use the udp bits? The http json bits have been deprecated.

I'm interested in the udp bits for no-impact logging of webapps.
